core/os/spin_lock.h

```diff
@@ -81,7 +81,7 @@ _ALWAYS_INLINE_ static void _cpu_pause() {
 	__builtin_ia32_pause();
 #elif defined(__arm__) || defined(__aarch64__) // ARM.
 	asm volatile("yield");
-#elif defined(__powerpc__) || defined(__ppc__) || defined(__PPC__) // PowerPC.
+#elif defined(__powerpc__) // PowerPC.
```

Comment:
I just simplified because `__powerpc__` should be sufficient here, `__ppc__` and `__PPC__` are legacy aliases for old hardware we can in no way aim to support.